packages/tables/resources/views/cells/image.blade.php

```diff
@@ -0,0 +1,21 @@
+<a
+    href="{{ $column->getValue($record) }}"
+    target="_blank"
+    @if ($column->cover)
+        class="block bg-center {{ $column->getClass() }}"
+        style="
+            {{ $column->getWidth() ? "width: {$column->getWidth()};" : '' }}
+            {{ $column->getHeight() ? "height: {$column->getHeight()};" : '' }}
+            background-image: url({{ $column->getValue($record) }});"
+    @endif
+>
+    @if (!$column->cover)
```

Comment:
Not sure what @danharrin thinks, but `@unless` is being used elsewhere in the code base. Might be good for consistency.

Reply:
Right, changed!